### PR TITLE
Visit 'Build' links instead of computing url manually and working around POST

### PR DESCRIPTION
**Pull request by Vlatombe**

### CODE REVIEW

**Comment by abayer** [comment]
:bee:

**Comment by olivergondza** [comment]
Causes some test failures. The idea look good to me.

**Comment by Vlatombe** [comment]
Looking at the test failures...

File: src/main/java/org/jenkinsci/test/acceptance/po/Job.java [hunk 1]
```diff
@@ -274,15 +274,12 @@ public Build scheduleBuild() {
     }
 
     public Build scheduleBuild(Map<String, ?> params) {
+        visit("");
```
Comment (olivergondza) [inline src/main/java/org/jenkinsci/test/acceptance/po/Job.java]: `open()`

**Comment by olivergondza** [comment]
:bee:

**Pull request merged**